pkg/agent/nodeportlocal/rules/iptable_rule.go

```diff
@@ -106,35 +107,29 @@ func (ipt *iptablesRules) AddAllRules(nplList []PodNodePort) error {
 	return nil
 }
 
-// DeleteRule deletes a specific NPL rule from NodePortLocalChain chain
-func (ipt *iptablesRules) DeleteRule(port int, podIP string) error {
+// DeleteRules deletes a specific NPL rule from NodePortLocalChain chain
+func (ipt *iptablesRules) DeleteRules(port int, podIP string) error {
```

Comment:
can we have consistent naming for the `port` parameter (consistent with `AddRule`)? looks like it should be `podPort`.

Reply:
Indeed. Updated.

Reply:
Took another look and I think actually it is `nodePort.
`buildRuleForPod` in `AddRule` explains it: https://github.com/antrea-io/antrea/blob/b0c0e9be05d848f2773621ea0b0df98862156306/pkg/agent/nodeportlocal/rules/iptable_rule.go#L84